checkov/common/images/image_referencer.py

```diff
@@ -159,10 +161,19 @@ def add_image_records(
         report_type: str,
     ) -> None:
         """Adds an image record to the given report, if possible"""
+        from checkov.common.bridgecrew.platform_integration import bc_integration
```

Comment:
it can't be imported at top level, because of circular dependencies, right? if yes, then please move it one function higher and pass it in as an argument.